### ISSUE TITLE
Suggestion: Add Screenshots

### ISSUE THREAD

**Issue by sbrl** [Runscope/requestbin#72]
Since an online demo is no longer available, a few screenshots (maybe in an imgur album?) would work wonders to show people how requestbin looks and functions.

**Comment by Beyarz** [comment]
Here's how the site looked like before it went down.
https://web.archive.org/web/20180130063422/https://requestb.in/

**Comment by sbrl** [comment]
Cool!

**Comment by nanoquanta** [comment]
I could only reach to this extent after trying for hours if i was using the postbin or using anything wrong that caused error. Heres a link 
https://web.archive.org/web/20170515161717/http://requestb.in/api/v1/bins

**Comment by nanoquanta** [comment]
Clicking on Beyarz link bought me to the page 
https://web.archive.org/web/20180130063422/https://requestb.in/
Is that the one? Then i clicked on create a requestbin which led me to a page that i posted on the previous comment. 
Is there anything I am doing wrong?

**Comment by Beyarz** [comment]
@nanoquanta No, it seems that accessing [requestb.in/api/v1/bins](http://requestb.in/api/v1/bins) is not possible through the archive.

**Comment by nanoquanta** [comment]
What do you mean?

**Comment by nanoquanta** [comment]
Here’s screenshot what i landed on after clicking the link you posted which is what one is expected to do looking at this post! Thats what I understand at least. And since you mentioned that I shouldn’t have access to that then i realized yeah, i don't trust that app that i used to post that link and just to post an imgur link, it took me this long. Hopefully these pictures give little if not much additional info. Thanks for the info and hope this helps. 

http://imgur.com/kJ0ruvk

**Comment by Beyarz** [comment]
The homepage is the only page I've managed to access through the archive, other urls doesn't offer anything other than a error message.

**Comment by nanoquanta** [comment]
I am confused and what does it mean?

**Comment by Beyarz** [comment]
> Clicking on Beyarz link bought me to the page
> https://web.archive.org/web/20180130063422/https://requestb.in/
> Is that the one? Then i clicked on create a requestbin which led me to a page that i posted on the previous comment.
> Is there anything I am doing wrong?

You are not doing anything wrong. That is the url.
